packages/core/src/admin-ui/router.tsx

```diff
@@ -27,8 +27,7 @@ export const Link = ({ href, as, replace, scroll, shallow, prefetch, ...props }:
       scroll={scroll}
       shallow={shallow}
       prefetch={prefetch}
-    >
-      <a {...props} />
-    </NextLink>
+      {...props}
```

Comment:
minor question - since we're removing the `<a>` tag here, do we need to destructure certain props out of `LinkProps` above? seems like this could be simplified to just be:
```
export const Link = NextLink;
```
🤔

Reply:
or even just `export Link from "next/link"` 🤔

Reply:
You are right, I just made the change.